lib/matplotlib/backends/web_backend/js/mpl.js

```diff
@@ -256,6 +260,18 @@ mpl.figure.prototype._init_canvas = function () {
         'mouseleave',
         on_mouse_event_closure('figure_leave')
     );
+    rubberband_canvas.addEventListener(
+        'touchmove',
+        on_touch_event_closure('touch_move')
+    );
+    rubberband_canvas.addEventListener(
+        'touchenter',
```

Comment:
Honestly I've not seen this event (nor touchleave) actually fire. Perhaps I don't have the right kind of device (mobile or chrome developer tools).